packages/dataviews/src/filters.js

```diff
@@ -63,19 +56,21 @@ const Filters = memo( function Filters( { fields, view, onChangeView } ) {
 				} );
 		}
 	} );
-
+	// Sort filters by primary property. We need the primary filters to be first.
+	filters.sort( ( a, b ) => b.isPrimary - a.isPrimary ); // Type coercion to numbers.
```

Comment:
We should consider sorting the filters alphabetically:

```suggestion
filters.sort( ( a, b ) => {
  if ( a.isPrimary && ! b.isPrimary ) {
    return -1;
  }

  if ( ! a.isPrimary && b.isPrimary ) {
    return 1;
  }

  return a.header.localeCompare(b.name);
});
```

Reply:
Sounds good.